tests/lib/rules/no-extra-label.js

```diff
@@ -39,15 +39,60 @@ ruleTester.run("no-extra-label", rule, {
         {code: "A: for (a of ary) { switch (b) { case 0: break A; } }", parserOptions: {ecmaVersion: 6}}
     ],
     invalid: [
-        {code: "A: while (a) break A;", errors: ["This label 'A' is unnecessary."]},
-        {code: "A: while (a) { B: { continue A; } }", errors: ["This label 'A' is unnecessary."]},
-        {code: "X: while (x) { A: while (a) { B: { break A; break B; continue X; } } }", errors: ["This label 'A' is unnecessary."]},
-        {code: "A: do { break A; } while (a);", errors: ["This label 'A' is unnecessary."]},
-        {code: "A: for (;;) { break A; }", errors: ["This label 'A' is unnecessary."]},
-        {code: "A: for (a in obj) { break A; }", errors: ["This label 'A' is unnecessary."]},
-        {code: "A: for (a of ary) { break A; }", errors: ["This label 'A' is unnecessary."], parserOptions: {ecmaVersion: 6}},
-        {code: "A: switch (a) { case 0: break A; }", errors: ["This label 'A' is unnecessary."]},
-        {code: "X: while (x) { A: switch (a) { case 0: break A; } }", errors: ["This label 'A' is unnecessary."]},
-        {code: "X: switch (a) { case 0: A: while (b) break A; }", errors: ["This label 'A' is unnecessary."]}
+        {
+            code: "A: while (a) break A;",
+            output: "A: while (a) break;",
+            errors: ["This label 'A' is unnecessary."]
+        },
+        {
+            code: "A: while (a) { B: { continue A; } }",
+            output: "A: while (a) { B: { continue; } }",
+            errors: ["This label 'A' is unnecessary."]
+        },
+        {
+            code: "X: while (x) { A: while (a) { B: { break A; break B; continue X; } } }",
+            output: "X: while (x) { A: while (a) { B: { break; break B; continue X; } } }",
+            errors: ["This label 'A' is unnecessary."]
+        },
+        {
+            code: "A: do { break A; } while (a);",
+            output: "A: do { break; } while (a);",
```

Comment:
That was covered in #5955. I'm closing this, since we have the same concern about typos as there.